drivers/SmartThings/zigbee-switch/src/multi-switch-no-master/init.lua

```diff
@@ -11,61 +11,114 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
+local capabilities = require "st.capabilities"
+local st_device = require "st.device"
+local clusters = require "st.zigbee.zcl.clusters"
+local OnOff = clusters.OnOff
 
+local parent_endpoint = 1
```

Comment:
@NikitaKuchta Please, could You use capital letters here ? (it would be noticeable as a constant value then)

Reply:
agree, done